api-samples/power/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "manifest_version": 3,
+
+  "name": "__MSG_extensionName__",
```

Comment:
Unfortunately the sample list generator doesn't have support for this, so I think we need to either add that or directly include the name and description here.

Reply:
Added support for it in cdde058f49028f4d9fcb7c0ab62256deb230bd7d

Reply:
Thanks! I'll leave a couple of comments but overall looks good :)